### DISCUSSION TITLE
Help! Error

### DISCUSSION THREAD

**Discussion by brunoberriel2001** [lllyasviel/Fooocus#2953] [Q&A] [answered]
Hello everyone,

I am having an issue when trying to run my program with the run.bat file. The program starts correctly but then it pauses and doesn't continue. Does anyone know what solutions might exist for this problem? I appreciate any help you can provide.

Thank you in advance.
![1](https://github.com/lllyasviel/Fooocus/assets/162738631/b76f52dc-ab91-4e4a-a61c-0a8e5f80f433)

![image](https://github.com/lllyasviel/Fooocus/assets/162738631/7ec4739a-a9d2-4a3a-b7ec-2ae64ce526a5)

**Comment by mashb1t** [comment, marked as answer]
Please follow the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) (especially the swap section)and check your resource allocation (disk, RAM, VRAM) in the task manager.
If this doesn't help please open a new issue and provide all necessary information.

**Reply by brunoberriel2001** [reply]
Thank you! Problem solved.

**Reply by PrathameshSolanke** [reply]
Hi , how did solve the problem I am getting the same error at first in cmd everything works perfectly and it opens a window in chrome and whenever I prompt something it does not show me any rendering process, being a non-programmer I don't understand that complex answer if you are ok please tell me step by step.

Thanks

**Reply by mashb1t** [reply]
> Please follow the troubleshooting guide (especially the swap section)

@PrathameshSolanke This is most likely going to solve your issue, but feel free to open an issue and provide all necessary information mentioned in the template.
